pages/k8s/charm-tigera-secure-ee.md

```diff
@@ -0,0 +1,109 @@
+---
```

Comment:
@Cynerva this is the source file!

Reply:
Thanks. I was looking at the one under `pages/k8s/_site/`. My bad.

Reply:
it's my bad really - those rendered pages were part of an interstitial phase which is no longer required so I cleaned them all out